test/acl.test.js

```diff
@@ -363,19 +363,18 @@ describe('security ACLs', function() {
 });
 
 describe('access check', function() {
-  var app;
-  before(function() {
-    app = loopback();
-    app.use(loopback.rest());
-    app.enableAuth();
-    app.dataSource('test', {connector: 'memory'});
-  });
-
   it('should occur before other remote hooks', function(done) {
-    var MyTestModel = app.model('MyTestModel', {base: 'PersistedModel', dataSource: 'test'});
+    var app = loopback();
+    var MyTestModel = app.registry.createModel('MyTestModel');
     var checkAccessCalled = false;
     var beforeHookCalled = false;
 
+    app.use(loopback.rest());
+    app.set('remoting', { errorHandler: { debug: true, log: false }});
```

Comment:
I am not sure as I see `errorHandler`in [`package.json`](https://github.com/strongloop/loopback/blob/2.x/package.json)  for 2.x.